src/index.css

```diff
@@ -189,3 +189,20 @@ button {
   background-color: transparent;
 }
 
+:root {
+  --white: #ffffff;
+  --black: #000000;
+  --turquoise: #24cca7;
+  --pink: #ff6596;
+  --lightpink: #ffd8d0;
+  --violet: #c5baff;
+  --blue: #4a56e2;
+  --lightBlue: #6e78e8;
+  --grey: #bdbdbd;
+  --lightGrey: #e0e0e0;
+  --darkGrey: #a6a6a6;
+  --background: #e7eaf2;
+  --box-shadow-table: -5px 6px 7px -3px rgba (0, 0, 0, 0.29);
+  --t-t-function: cubic-bezier(0.4, 0, 0.2, 1);
+  --t-duration: 250ms;
+}
```

Comment:
root css jest w pliku variables.css w folderze stylesheet. To jeśli możesz to proszę przerzuć to tam, sprawdź czy się nie duplikują i stamtąd wyciągaj :)